requirements.txt

```diff
@@ -0,0 +1 @@
+pygpio
```

Comment:
```suggestion
pigpio
```

Reply:
Agree, will check version of the Daemon and make the package match since daemon updates require apt usage most likely.

Reply:
Since we're changing the install instructions in the readme to install pigpio form the source, we should freeze this at the same version as the source we're installing from: https://github.com/joan2937/pigpio/releases/tag/v79